web/html/src/components/icontag.js

```diff
@@ -85,13 +85,15 @@ function IconTag(props) {
     "item-download-csv": "fa spacewalk-icon-download-csv",
     "item-edit": "fa fa-edit",
     "item-enabled": "fa fa-check text-success",
+    "item-enabled-pending": "fa fa-check text-warning",
     "item-import": "fa fa-level-down",
     "item-search": "fa fa-eye",
     "item-ssm-add": "fa fa-plus-circle",
     "item-ssm-del": "fa fa-minus-circle",
     "item-upload": "fa fa-upload",
     "item-order": "fa fa-sort",
     "item-error": "fa fa-times text-danger",
+    "item-error-pending": "fa fa-times text-warning",
```

Comment:
Pretty much same as before, but in case of *errors* we should use the red color: I'd suggest to use `fa fa-hand-o-right text-error` in this case.

Reply:
Done. Please see screenshots.